### ISSUE TITLE
About object boundingbox label.

### ISSUE THREAD

**Issue by deepbrainme** [alanlukezic/d3s#1]
I have clone your code and vos dataset(version2018), but I came across a run error.
It show me that my dataset don't  have a file in  named 'groundtruth-obj_id.txt',  in each seqence in youtube-vos dataset.   My dataset have some wrong or miss some files ?  

 def _read_anno(self, seq_path, obj_id):
        anno_file = os.path.join(seq_path, **"groundtruth-%s.txt" % obj_id**)
        gt = pandas.read_csv(anno_file, delimiter=',', header=None, dtype=np.float32, na_filter=False, low_memory=False).values
        return torch.tensor(gt)

**Comment by alanlukezic** [comment]
Thank you for pointing this out, I forgot to include some files. The VOS dataset does not have bounding boxes, so I generated them from segmentation masks. You can download them on the link below and copy them in the dataset. Then it should work.
Bounding boxes download link: http://data.vicos.si/alanl/d3s/rectangles.zip

**Comment by deepbrainme** [comment]
Thanks for your reply ,I copy recatngles.zip to train/Annotations/ ,and unzip it. Now it works.
